docs/source/technical_overview.rst

```diff
@@ -107,7 +107,7 @@ operating on objects of the following types:
     .. |apply2|            replace:: :meth:`~pymor.operators.interface.Operator.apply2`
     .. |apply_inverse|     replace:: :meth:`~pymor.operators.interface.Operator.apply_inverse`
     .. |assembled|         replace:: :meth:`assembled <pymor.operators.interface.Operator.assemble>`
-    .. |_assemble_lincomb| replace:: :meth:`~pymor.operators.interface.Operator._assemble_lincomb`
+    .. |_assemble_lincomb| replace:: :meth:`!~pymor.operators.interface.Operator._assemble_lincomb`
```

Comment:
Is the problem, that this is a private method?

Reply:
Yes, I guess private methods also don't appear in the documentation.

Reply:
Is there may be some way to manually include some methods? For instance this one is kind of an interface method that should be documented in principle.

Reply:
It looks like it is possible via `autoapi-skip-member` functionality.